### ISSUE TITLE
Come up with a consistent set of genes

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/clin.iobio#391]
Currently, iobio uses a combination of RefSeq and GenCode. Mosaic uses Ensembl. We need to use HGNC.

Determine the best set of genes (which must contain HGNC) to be sufficient. Is Ensembl + HUGO sufficient?

**Comment by mvelinder** [comment]
Run clin.iobio with demo data. Jump to Select phenotypes step. Manually add gene ARPC3P3. Go to Review variants step and get an error about the gene not being recognized. I found it by actually scrolling through the drop down of possible genes. So it was supposedly a gene name that iobio already recognized. I'm sure there are others like this too, I just happened to bump into this one randomly in testing clin

![image](https://user-images.githubusercontent.com/22081541/148429273-73f69178-dfc2-4490-bb2e-0b3e5fb9f758.png)
![image](https://user-images.githubusercontent.com/22081541/148429290-a18848f2-265a-48b9-bd84-177431024025.png)

In gene.iobio you can get suggested the gene name from the dropdown, select it, it will show up in the left menu, not analyze it, but then when you click the name in the left menu it gives the unable to find transcripts error

![image](https://user-images.githubusercontent.com/22081541/148429309-4fc91fd3-9d34-44f6-bc57-4692f73c0c3b.png)
![image](https://user-images.githubusercontent.com/22081541/148429325-a58ee724-99e2-4aec-b6b5-8102d8406946.png)
![image](https://user-images.githubusercontent.com/22081541/148429336-af6fabdf-eb86-4ce6-8123-f9fea1e0b7e5.png)

Same issue as above, with no comprehensive gene name set. But was another random example

**Comment by AlistairNWard** [comment]
Yeah, this is a bad error message - the gene isn't unknown, it doesn't have any transcripts. Then the second problem is allowing it to be entered, but only providing an error when you subsequently click on the gene name. We need a better way of handling all this stuff in general. If we don't have transcripts, we can't analyse, but we should probably keep the gene names in the list with an icon and tooltip explaining that they can't be analyzed without known transcripts.

If there are other sources of transcripts, that would also be a good thing to look at.

**Comment by mvelinder** [comment]
Not sure about the transcript source in iobio, but there is an Ensembl transcript for this gene here https://uswest.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000256745;r=11:94188449-94188997;t=ENST00000397300

**Comment by AlistairNWard** [comment]
A solid 1 exon transcript! How often do we update the gff3 file we pull transcripts from?
